data/SPRING/projects.yaml

```diff
@@ -142,3 +142,31 @@ projects:
         date_updated: 2020-02-21
         maturity: 2
             
+    SwissCovid:
+        name: SwissCovid Exposure Notification
+        description: A privacy preserving exposure notification protocol to inform people about evenutal exposition to the CoVid-19 virus
```

Comment:
I would not use the word "eventual" here (it is a "faux ami" with the French "éventuel"), unless you mean that all people will sooner or later be exposed. ;-)  → "about possible exposure"

Also, I think the official spelling is "COVID-19".